<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MTcpListenWidget</class>
 <widget class="QWidget" name="MTcpListenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btStart">
       <property name="text">
        <string>&amp;listen</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btStop">
       <property name="text">
        <string>&amp;stop</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cbForce">
       <property name="text">
        <string>f&amp;orce</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <resources/>
 <connections/>
</ui>
